pepys_import/resolvers/command_line_resolver.py

```diff
@@ -192,11 +192,12 @@ def resolve_reference(
                 .all()
             )
         elif db_class.__tablename__ == constants.PRIVACY:
-            objects = data_store.session.query(db_class).order_by(desc(db_class.level)).all()
```

Comment:
This is very minor @BarisSari , but wouldn't it be (very, very, very slightly) more efficient to use `.limit(7)`, than to use `.all()` then trim the first 7 values?

Reply:
Yes, it's more efficient but I didn't want to fetch the table again for showing it in 'add a new privacy' option. That's why I fetched all values and added the first 7 values to the options. If a user selects 'add a new privacy', it then shows all values in the table without sending another query to the database.

Reply:
Aah yes, very good :-)